src/components/screens/admin-vacation-management/NotificationSnackbar.tsx

```diff
@@ -0,0 +1,31 @@
+import type React from 'react';
+import { Snackbar, Alert } from '@mui/material';
+
+interface NotificationSnackbarProps {
```

Comment:
I think this component might be a bit unnecessary. We already have a global error handler set up with atoms that you should use for any errors/ catch blocks (See setError usage).

If howvever you want to keep this the severity should be typed using an enum rather than just strings.